projects/demo/src/pages/recipes/card/examples/1-basic/style.less

```diff
@@ -0,0 +1,7 @@
+.cvv {
+    max-width: 5rem;
+}
+
```

Comment:
On 16'' screen it looks in the following way:
<img width="1787" alt="Screenshot 2023-01-30 at 11 42 21" src="https://user-images.githubusercontent.com/35179038/215428751-c97390f8-13db-4744-9a19-1e4061ade0e9.png">

Reply:
What do you think about to add `max-width: 30rem` for wrapper ?

<img width="1787" alt="Screenshot 2023-01-30 at 11 44 07" src="https://user-images.githubusercontent.com/35179038/215429094-6fe57f16-94fc-41cf-85f7-00f2c0254915.png">